hack/.golint_failures

```diff
@@ -0,0 +1,33 @@
+pkg/activator
```

Comment:
I'm not a fan of this package whitelisting approach. You're hiding the errors from the whitelisted packages (and there are a lot here), so nobody has any signal or incentive to fix them (unless one explicitly creates a linting PR).